spec/amber/cli/commands/generator_spec.cr

```diff
@@ -192,7 +192,6 @@ module Amber::CLI
           File.read("./src/handlers/authenticate.cr").should contain camel_case
           File.read("./src/handlers/authenticate.cr").should contain snake_case
           File.read("./src/models/admin_user.cr").should contain class_definition_prefix
-          File.read("./src/models/admin_user.cr").should contain snake_case
```

Comment:
What changed to make this no longer necessary/valid?

Reply:
Granite models aren't using `snake_case` anymore, see https://github.com/amberframework/granite-orm/pull/117

This was a **breaking change**, next time we should release a major (0.X.0) version 😅

Reply:
oops.  shoot.  your right. I suck.  my bad.